docs/sources/reference/run.md

```diff
@@ -552,6 +553,22 @@ would be 2*300M, so processes can use 300M swap memory as well.
 We set both memory and swap memory, so the processes in the container can use
 300M memory and 700M swap memory.
 
+By default, processes in a container are killed if an out-of-memory (OOM) occurs.
+To disable this behaviour, use the `--oom-kill-disable` option. Disabling the OOM killer
+should only be done on containers that have the `-m/--memory` option set.
+
+Examples:
+
+    $ docker run -ti -m 100M --oom-kill-disable ubuntu:14.04 /bin/bash
```

Comment:
@thaJeztah I was afraid of this; wow, I think that is confusing even without the lemon drops I've been drinking.  Uh, ok. I'll tweak my comments. We do need to fix this though, I think you are right.